src/vos/evtree.c

```diff
@@ -2721,7 +2721,7 @@ evt_ent_array_fill(struct evt_context *tcx, enum evt_find_opc find_opc,
 						DF_RECT" overlaps with "DF_RECT
 						"\n", DP_RECT(rect),
 						DP_RECT(&rtmp));
-					rc = -DER_NO_PERM;
+					rc = -DER_VOS_PARTIAL_UPDATE;
```

Comment:
now for failed rebuild the half-rebuilt data will be discarded, so how could the vos partial update hit again?

Reply:
there are not enough log for me to understand this yet, but "retry" can fix this this. I will collect more log to understand this.

My suspicion is due to enumeration limit.